lexicons/com/atproto/identity/getRecommendedDidCredentials.json

```diff
@@ -0,0 +1,28 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.getRecommendedDidCredentials",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Describe the credentials that should be included in the DID doc of an account that is migrating to this service.",
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "rotationKeys": {
```

Comment:
it is a tiny bit weird having `rotationKeys` here. it matches the PLC operation syntax/fields, but the other fields here match DID document fields but this one doesn't. I guess it's fine? `plcRotationKeys` is tempting but probably not worth breaking consistency.

a `description` on this field (in the Lexicon schema) would be good.

Reply:
Yeah I agree :/ But I think it makes most sense to return them here & keep it consistent with the other routes.

One thing we can do that may make it go down a bit easier is not return `rotationKeys` when this route is requested by a `did:web`. If we add some additional method that requires another field, we can add another property to the response this would be undefined for plc & web dids

Reply:
nice! that is a good way to handle it.